packages/braid-design-system/lib/components/Text/Text.screenshots.tsx

```diff
@@ -5,40 +5,70 @@ import { Text, Stack, Column, Columns, IconPositive } from '../';
 // TODO: COLORMODE RELEASE
 // Use public import
 import { Box } from '../Box/Box';
-import { textSizeUntrimmed as textSizes } from '../../css/typography.css';
+import { textSizeUntrimmed, fontWeight } from '../../css/typography.css';
 import { backgrounds, textAlignments } from '../../utils/docsHelpers';
 
 const Container = ({ children }: { children: ReactNode }) => (
   <div style={{ maxWidth: '300px' }}>{children}</div>
 );
 
+const textSizes = ['large', 'standard', 'small', 'xsmall'] as Array<
+  keyof typeof textSizeUntrimmed
+>;
+const textWeights = ['strong', 'medium', 'weak'] as Array<
+  keyof typeof fontWeight
+>;
```

Comment:
I'm not sure how much value these typecasts are providing. The `textSizeUntrimmed` object could only contain `large` as a key, and you'd end up with an array of type `"large"[]`, which is not the correct type, but would still pass type-checking.

You're already passing these text sizes into the `size` prop of `Text` anyway, which should catch any bad keys sneaking through I feel like `as const` would work just fine here, and won't result in a potentially incorrect type.
```suggestion
const textSizes = ['large', 'standard', 'small', 'xsmall'] as const;
const textWeights = ['strong', 'medium', 'weak'] as const;
```

Reply:
The intent is that if any additional sizes/weights were added in the future this would break and require you to add the new key in the correct position within the array. Realising that this wouldnt work as intended though.

Only reason im creating this array is to control the visual order of the variants.